it/test/testModels/DataStoreProfile.scala

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package testModels
+
+import play.api.libs.json.{OFormat, OWrites, Reads, __}
+
+case class DataStoreProfile(
+  eori: String,
+  actorId: String,
+  ukimsNumber: String,
+  nirmsNumber: Option[String],
+  niphlNumber: Option[String]
+)
+
+object DataStoreProfile {
+
+  val reads: Reads[DataStoreProfile] = {
+
+    import play.api.libs.functional.syntax._
+
+    (
+      (__ \ "eori").read[String] and
```

Comment:
just send actorId, eori will be present in the path

Reply:
This is just for the testing - so we can bring back the right shaped data